workspaces/global-header/docs/index.md

```diff
@@ -2,7 +2,8 @@
 
 Red Hat Developer Hub includes a new configable and highly extendable global header plugin starting with RHDH 1.5.
 
-By default it includes a Search input field, Create, Support[^1] and Notifications[^2] icon buttons and a user profile dropdown.
+By default it includes a Search input field, Create, Starred[^1], Support[^2] and Notifications[^3] icon buttons and a user profile dropdown.
 
-[^1]: Only when the Support URL is configured in the `app-config.yaml`.
-[^2]: Only when the notification plugin is installed.
+[^1]: Only when an enitity is starred.
+[^2]: Only when the Support URL is configured in the `app-config.yaml`.
+[^3]: Only when the notification plugin is installed.
```

Comment:
```suggestion
[^3]: Only when the notifications plugin is installed.
```
I know this is from the old code, but let's correct it here, thank you 💮

Reply:
Done.